Sources/CvUnitAI.h

```diff
@@ -528,6 +528,7 @@ class CvUnitAI : public CvUnit
 	bool isWaitingOnUnitAIAny();
 	void setWaitingOnUnitAIAny();
 	bool AI_isNegativePropertyUnit();
+	int getMyAggression(int iAttackProb);
```

Comment:
Last thing: it should be a const function as it does not modify the object members.
`int getMyAggression(int iAttackProb) const;`
also add const in the cpp in the same way.

Reply:
Now I get 
error C2511: 'void CvUnitAI::AI_animalMove(void) const' : overloaded member function not found in 'CvUnitAI'

What did I do wrong?

Reply:
ah lol, I'm tired, added const to wrong function in cpp.